api/controllers/shabads.js

```diff
@@ -261,17 +261,21 @@ exports.search = async (req, res) => {
 exports.resultsInfo = async (req, res) => {
   const { VerseID } = req.params;
```

Comment:
Can we change it to `verseIDs`?

Reply:
Changed it to VerseIds to keep it same with existing code.